rotkehlchen/history/events.py

```diff
@@ -355,8 +354,8 @@ def new_step_cb(state_name: str) -> None:
                 fail_callback=fail_history_cb,
                 new_step_data=(new_step_cb, exchange.name),
             )
-            step = self._increase_progress(step, total_steps)
-            step = step_before + STEPS_PER_CEX  # Make sure remote failures don't throw steps off
+            # each exchange instance executes STEPS_PER_CEX steps for total_steps
+            step = self._increase_progress(step, total_steps, step_by=STEPS_PER_CEX)
```

Comment:
Why a new argument is needed? Why not do it this way?

```suggestion
            step = self._increase_progress(step + STEPS_PER_CEX, total_steps)
```

Reply:
In `_increase_progress` the `step` variable will be modified and the new value will be `step + STEPS_PER_CEX + 1`. To achieve what you want then the correct value would be `step + STEPS_PER_CEX - 1` and this starts to be ugly. `step_by` seems a more redeable argument for me. What I missed is a docstring explaining what it does

Reply:
Hmmm yeah, probably makes sense :+1: